toolkit/scripts/pkggen.mk

```diff
@@ -233,6 +233,7 @@ $(preprocessed_file): $(cached_file) $(go-graphPreprocessor)
 
 ######## PACKAGE BUILD ########
 
+cache_archive	= $(OUT_DIR)/cache.tar.gz
```

Comment:
Should we be modifying the toolkit via fasttrack route ?

Reply:
This change is intended to make improvements in the fast-track builds, thus this route. A good question, though, fast-track is in general intended for everything that needs to be pushed out quickly. In this case that's making fast-track faster, so that we can push out other things quicker.;)